examples/benchmark.py

```diff
@@ -43,32 +43,46 @@
     "rgbd_semantic": {"depth_sensor": True, "semantic_sensor": True},
 }
 
-# resolutions = [128] # (debug)
+#  resolutions = [128] # (debug)
 resolutions = [128, 256, 512]
+nprocs_tests = [1, 3, 5]
 
-performance = []
-for resolution in resolutions:
-    default_settings["width"] = default_settings["height"] = resolution
-    perf = {}
-    for key, value in benchmark_items.items():
-        print(" ---------------------- %s ------------------------ " % key)
-        settings = default_settings.copy()
-        settings.update(value)
-        perf[key] = demo_runner.benchmark(settings).get("fps")
-        print(
-            " ====== FPS (%d x %d, %s): %0.1f ======"
-            % (settings["width"], settings["height"], key, perf[key])
-        )
-    performance.append(perf)
+performance_all = {}
+for nprocs in nprocs_tests:
+    default_settings["num_processes"] = nprocs
+    performance = []
+    for resolution in resolutions:
+        default_settings["width"] = default_settings["height"] = resolution
+        perf = {}
+        for key, value in benchmark_items.items():
+            demo_runner = dr.DemoRunner(default_settings, dr.DemoRunnerType.BENCHMARK)
+            print(" ---------------------- %s ------------------------ " % key)
+            settings = default_settings.copy()
+            settings.update(value)
+            perf[key] = demo_runner.benchmark(settings).get("fps")
+            print(
+                " ====== FPS (%d x %d, %s): %0.1f ======"
+                % (settings["width"], settings["height"], key, perf[key])
+            )
+        performance.append(perf)
+
+    performance_all[nprocs] = performance
 
-print(" ================ Performance (FPS) ===========================================")
-title = "Resolution "
-for key, value in perf.items():
-    title += "%15s" % key
-print(title)
-for idx in range(len(performance)):
-    row = "%d x %d" % (resolutions[idx], resolutions[idx])
-    for key, value in performance[idx].items():
-        row += "%15.1f" % value
-    print(row)
-print(" ==============================================================================")
+for nproc, performance in performance_all.items():
+    print(
+        " ================ Performance (FPS) NPROC={} ===================================".format(
+            nproc
+        )
+    )
+    title = "Resolution "
+    for key, value in perf.items():
+        title += "%15s" % key
+    print(title)
+    for idx in range(len(performance)):
+        row = "%d x %d" % (resolutions[idx], resolutions[idx])
+        for key, value in performance[idx].items():
+            row += "%15.1f" % value
+        print(row)
+    print(
+        " =============================================================================="
+    )
```

Comment:
Can you fix the spacing to have the titles more centered? Right now it looks like 
```
 ================ Performance (FPS) NPROC=1 ===================================
Resolution             rgb           rgbd     depth_only  semantic_only  rgbd_semantic
128 x 128         4288.3         2052.2         4266.9          540.3          429.1
256 x 256         3162.0         1704.4         3245.0          518.6          387.3
512 x 512         1611.6          803.4         1614.5          451.1          280.9
```

But should look like 
```
 ================ Performance (FPS) NPROC=1 ===================================
Resolution         rgb            rgbd        depth_only    semantic_only  rgbd_semantic
128 x 128         4288.3         2052.2         4266.9          540.3          429.1
256 x 256         3162.0         1704.4         3245.0          518.6          387.3
512 x 512         1611.6          803.4         1614.5          451.1          280.9
 ==============================================================================
 ```
Or
```
 ================ Performance (FPS) NPROC=1 ===================================
Resolution           rgb           rgbd     depth_only  semantic_only  rgbd_semantic
128 x 128         4288.3         2052.2         4266.9          540.3          429.1
256 x 256         3162.0         1704.4         3245.0          518.6          387.3
512 x 512         1611.6          803.4         1614.5          451.1          280.9
 ==============================================================================
 ```
Or
```
 ================ Performance (FPS) NPROC=1 ===================================
Resolution        rgb            rgbd           depth_only      semantic_only  rgbd_semantic
128 x 128         4288.3         2052.2         4266.9          540.3          429.1
256 x 256         3162.0         1704.4         3245.0          518.6          387.3
512 x 512         1611.6          803.4         1614.5          451.1          280.9
 ==============================================================================
 ```